client/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import styles from "./SideNav.module.css";
+import Logo from "./../../assets/logo.svg";
+
+export default function SideNav() {
+  const [search, setSearch] = useState("");
+
+  const handleSearch = (value: string) => {
+    setSearch(value);
+  };
+  return (
+    <div className={styles.sidenav}>
+      <div className={styles["logo-con"]}>
+        <Logo className={styles.logo} />
+        <p className={styles["logo-text"]}>cascarita</p>
+      </div>
+      <div className={styles["input-icons"]}>
+        <i className="fa fa-search icon"></i>
+        <input
+          type="text"
+          placeholder="Search for..."
+          onChange={(e) => handleSearch(e.target.value)}
+          value={search}
+        />
+      </div>
+      <button>Home</button>
+      <button>Users</button>
+      <button>Team</button>
+      <button>League</button>
+      <button>Calendar</button>
+      <button>Settings</button>
+    </div>
+  );
```

Comment:
so overall it would look something like this. Also additionally keep in mind to use semantic html meaning 
Use `<nav>` for your navigation container.
Consider using `<ul>` and `<li>` for listing your navigation buttons.
```suggestion
return (
    <nav className={styles.sidenav}>
      <div className={styles["logo-con"]}>
        <Logo className={styles.logo} />
        <p className={styles["logo-text"]}>cascarita</p>
      </div>
      <SearchInput onSearch={setSearch} />
      <ul>
        {['Home', 'Users', 'Team', 'League', 'Calendar', 'Settings'].map((item) => (
          <li key={item}><NavButton>{item}</NavButton></li>
        ))}
      </ul>
    </nav>
  );
```

Reply:
Ack!